profiler/test/Datadog.Profiler.Native.Tests/ConfigurationTest.cpp

```diff
@@ -153,9 +153,9 @@ TEST(ConfigurationTest, CheckDefaultPprofDirectoryWhenVariableIsNotSet)
     auto configuration = Configuration{};
     auto expectedValue =
 #ifdef _WINDOWS
-        WStr("C:\\ProgramData\\Datadog-APM\\Pprof-files\\DotNet");
+        WStr("C:\\ProgramData\\SignalFx .NET Tracing\Pprof-files\\DotNet");
```

Comment:
```suggestion
        WStr("C:\\ProgramData\\SignalFx .NET Tracing\\Pprof-files\\DotNet");
```

also this files misses the "Modified by Splunk" comment

Reply:
As all/most others cpp files ;)